src/modules/product/product.service.js

```
@@ -320,38 +320,27 @@ export const updateProductService = async (id, updateData) => {
-    
-    // Update the product with validated data
-    const updatedProduct = await Product.findByIdAndUpdate(
-      id,
-      { $set: validatedData },
-      { new: true, runValidators: true }
-    ).populate('category');
+  // Update the product directly without validation
+  const updatedProduct = await Product.findByIdAndUpdate(
+    id,
+    { $set: processedData },
+    { new: true, runValidators: false }
```

Biome (1.9.4)

[error] Can't assign processedData because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)